### PR TITLE
fix: Course outline tests

### PR DESCRIPTION
**Pull request by vladislavkeblysh**
- Fixed status bar tests

### CODE REVIEW

File: src/course-outline/status-bar/StatusBar.test.jsx [hunk 1]
```diff
@@ -1,14 +1,26 @@
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { AppProvider } from '@edx/frontend-platform/react';
 
+import { initializeMockApp } from '@edx/frontend-platform';
 import StatusBar from './StatusBar';
 import messages from './messages';
+import initializeStore from '../../store';
```
Comment (PKulkoRaccoonGang) [inline src/course-outline/status-bar/StatusBar.test.jsx:9]:
```suggestion
import { AppProvider } from '@edx/frontend-platform/react';
import { initializeMockApp } from '@edx/frontend-platform';

import initializeStore from '../../store';
import StatusBar from './StatusBar';
import messages from './messages';
```

Comment (vladislavkeblysh) [reply src/course-outline/status-bar/StatusBar.test.jsx]: Done

File: src/course-outline/enable-highlights-modal/EnableHighlightsModal.test.jsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ describe('<EnableHighlightsModal />', () => {
 
     const submitButton = getByRole('button', { name: messages.submitButton.defaultMessage });
     fireEvent.click(submitButton);
-    expect(onEnableHighlightsSubmitMock).toHaveBeenCalledTimes();
+    expect(onEnableHighlightsSubmitMock).toHaveBeenCalledTimes(1);
```
Comment (ruzniaievdm) [inline src/course-outline/enable-highlights-modal/EnableHighlightsModal.test.jsx:45]: optional: `expect(onEnableHighlightsSubmitMock).toHaveBeenCalled();`

**Review by ruzniaievdm — APPROVED** [review]

File: src/course-outline/status-bar/StatusBar.test.jsx [hunk 3]
```diff
@@ -1,14 +1,26 @@
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { initializeMockApp } from '@edx/frontend-platform';
 
 import StatusBar from './StatusBar';
 import messages from './messages';
+import initializeStore from '../../store';
```
Comment (PKulkoRaccoonGang) [inline src/course-outline/status-bar/StatusBar.test.jsx:9]:
```suggestion
import initializeStore from '../../store';
import StatusBar from './StatusBar';
import messages from './messages';
```

**Pull request merged**